x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/translations.ts

```diff
@@ -83,3 +83,12 @@ export const COMMENTS_SECTION_TITLE = (comments: number) =>
     values: { comments },
     defaultMessage: 'Add comments ({comments})',
   });
+
+export const ADD_RULE_EXCEPTION_FROM_ALERT_COMMENT = (alertId: string) =>
+  i18n.translate(
+    'xpack.securitySolution.ruleExceptions.addExceptionFlyout.addRuleExceptionFromAlertComment',
+    {
+      values: { alertId },
+      defaultMessage: 'The conditions are populated from an Alert with id: ({alertId})',
```

Comment:
@elastic/security-docs could you help with the wording here? We wanted to prepopulate a comment for the user that notes what alert this exception makes reference to.